modules/cnv-cdrom-wizard-fields-web.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv-create-vms.adoc
+// * cnv/cnv_vm_templates/cnv-creating-vm-template.adoc
+
+[id="cnv-cdrom-wizard-fields-web_{context}"]
+= CD-ROM fields
+
+|===
+|Source | Description
+
+|Container
+|Specify the container path. For example: `kubevirt/fedora-registry-disk: latest`.
+
+|URL
+|Specify the URL path and size in GiB. Then, select the storage class for this URL from the drop-down list.
+
+|Attach Disk
+|Select the desired virtual machine disk that you want to attach.
```

Comment:
@lmandavi This still says "desired" -- please remove it

Reply:
Thanks for the catch, removed.

Reply:
https://cnv-3580--ocpdocs.netlify.com/openshift-enterprise/latest/cnv/cnv_virtual_machines/cnv-create-vms.html#cnv-cdrom-wizard-fields-web_cnv-create-vms